proto/uber/cadence/api/v1/service_visibility.proto

```diff
@@ -66,6 +69,23 @@ message ListWorkflowExecutionsResponse {
   bytes next_page_token = 2;
 }
 
+message ListAllWorkflowExecutionsRequest {
+  string domain = 1;
+  int32 maximum_page_size = 2;
```

Comment:
this is called `page_size` in `ListWorkflowExecutionsRequest` so let's stick with it for consistency